logstash-core/api/lib/app/root.rb

```diff
@@ -0,0 +1,15 @@
+# encoding: utf-8
+require "app"
+
+module LogStash::Api
+  class Root < BaseApp
+
+    get "/" do
+      content = { "name" => "Logstash API",
+                  "version" => { "number" => "0.1.0" },
+                  "tag" => "You know, for logs" }
```

Comment:
Logstash is more than just logs, so let's remove this :) I think I put it in the design doc, but we can pretty much remove it